app/services/participants/query.rb

```diff
@@ -37,7 +37,13 @@ def where_lead_provider_is(lead_provider)
     def where_updated_since(updated_since)
       return if ignore?(filter: updated_since)
 
-      scope.merge!(User.where(updated_at: updated_since..))
+      query1 = User.where(updated_at: updated_since..)
+      query2 = User.where(applications: { updated_at: updated_since.. })
+      query3 = User.where(participant_id_changes: { updated_at: updated_since.. })
+
+      query = query1.or(query2).or(query3)
```

Comment:
the tests aren't quite covering these queries here; if you omit query1 or query2 they still pass:

```
query = query1.or(query3)
```

```
query = query2.or(query3)
```

Reply:
This is due to the touch model, when application is updated, it updates user also.
I'll modify the tests to account for this.

Reply:
all tests are still passing when you omit the `users_updated_since` scope:

```
scope.merge!(applications_updated_since.or(participant_id_changes_updated_since))
```